library/common/config_template.cc

```diff
@@ -74,14 +74,12 @@ const char* config_template = R"(
       thresholds:
         - priority: DEFAULT
           # n.b: with mobile clients there are scenarios where all concurrent requests might be
-          # retries (e.g., when the phone goes offline). Therefore, Envoy Mobile allows all concurrent
-          # requests to be retries by setting the concurrent retry budget to 100%.
+          # retries (e.g., when the phone goes offline). Therefore, Envoy Mobile allows for the same
+          # amount of concurrency for retries as the default value for concurrency for requests.
           # This configuration uses the default for max concurrent requests (1024) because there is
           # no reasonable scenario where a mobile client should have even close to that many concurrent
           # requests.
-          retry_budget:
-            budget_percent:
-              value: 100
+          max_retries: 1024
```

Comment:
Why not just add a `min_retry_concurrency`? See:
https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/cluster/circuit_breaker.proto#cluster-circuitbreakers-thresholds-retrybudget

Reply:
If you just say `min_retry_concurrency: 1024` you'll then just have that be the lower bound.

Reply:
@tonya11en im not sure I see the benefit of having a retry budget if the min is the same as the max requests circuit breaker. In that case the concurrency would be capped at 1024 anyway you dice it. 

What would be interesting for this scenario would be a retry budget based on active _downstream_ requests. Because what we want here is to have active retries for potentially 100% of active downstream requests.

Reply:
Thanks for the reply below @tonya11en :).

Any thoughts about retry budgets based on number of concurrent downstream requests?